sim/lib/town_data.py

```diff
@@ -331,7 +331,7 @@ def generate_sites(bbox, query_files, sites_path, site_based_density_file=None):
             # read sites latitude and longitude
             locs_to_add=[]
             for site in data['elements']:
-                if s_type == "office" and spmkt == True:
```

Comment:
Previously shops are regarded as offices, so we need to exclude supermarkets from "shop offices". Now we need to do this for retail stores instead of offices.